packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -231,6 +231,12 @@ class _InternalAbstractMeshDataInfo {
      * We use that as a clue to force the material to sideOrientation = null
      */
     public _sideOrientationHint = false;
+
+    /**
+     * @internal
+     * if this is set to true, the mesh will be visible only if its parent(s) are also visible
+     */
+    public _inheritedVisibility = false;

```

Comment:
sorry I was unclear: I meant to move it as part of  _internalAbstractMeshDataInfo